cfgov/jinja2/v1/_includes/organisms/mega-menu.html

```diff
@@ -60,11 +60,11 @@
     <li class="list_item
                {{ _classes( nav_depth, '-item' ) }}"
         {{ 'data-js-hook=flyout-menu' if nav_children | count > 0 else '' }}>
-        {# TODO: Remove nav_depth < 2 check on has-children class when 3rd level transitions are in. #}
+        {# TODO: disable link (or overview link) of page user is currently on. #}
```

Comment:
Didn't this already exist with the green bar to the left? I think it was the "active_nav_item" or something similar